tests/test_python_api.py

```diff
@@ -67,10 +67,10 @@ def test_tcpwer():
     # There is no motivation for the selected combinations.
     # Hint: We recommend to use the defaults.
     for h_pwt, r_pwt, errors in [
-        ['character_based_points', 'character_based', 40],
+        ['character_based_points', 'character_based', 42],
```

Comment:
I also fixed this in the tcORC-WER PR. I hope that this doesn't cause merge conflicts

Reply:
Hmm. We should try to remove the red badge in the readme. I was not sure, when you plan to merge tcorc.

Reply:
I could merge it now and open another PR for any further changes to the tcORC-WER